ank/src/cli_commands/workload_table.rs

```diff
@@ -0,0 +1,315 @@
+            default_table,
+            WorkloadTableRow::ADDITIONAL_INFO_POS,
+            remaining_terminal_width,
+            Self::ADDITIONAL_INFO_SUFFIX,
+        );
+
+        Some(truncated_table.to_string())
+    }
+
+    // [impl->swdd~cli-shall-present-workloads-as-table~1]
+    pub fn create_table_wrapped_additional_info(&self) -> Option<String> {
+        let default_table = Self::default_table(&self.rows);
+
+        let total_table_width: usize = default_table.total_width();
+        let remaining_terminal_width =
+            self.terminal_width_for_additional_info(total_table_width)?;
+
+        let wrapped_table = Self::wrap_table_column(
+            default_table,
+            WorkloadTableRow::ADDITIONAL_INFO_POS,
+            remaining_terminal_width,
+        );
+
+        Some(wrapped_table.to_string())
+    }
+
+    fn default_table(rows: &Vec<RowType>) -> Table {
+        let mut table = Table::new(rows);
+        let basic_table = table.with(Style::blank()).to_owned();
+
+        Self::set_custom_table_padding(basic_table)
+    }
+
+    fn set_custom_table_padding(mut table: Table) -> Table {
+        let first_column_default_padding =
+            table
+                .get_config()
+                .get_padding(tabled::grid::config::Entity::Column(
+                    WorkloadTableRow::FIRST_COLUMN_POS,
+                ));
+
+        let last_column_default_padding =
+            table
+                .get_config()
+                .get_padding(tabled::grid::config::Entity::Column(
+                    WorkloadTableRow::ADDITIONAL_INFO_POS,
+                ));
+
+        /* Set the left padding of the first and the right padding of the last column to zero
+        to align the table content to the full terminal width for better output quality. */
+        const ZERO_PADDING: usize = 0;
+        table
+            .with(Modify::new(Columns::first()).with(Padding::new(
+                ZERO_PADDING,
+                first_column_default_padding.right.size,
+                first_column_default_padding.top.size,
+                first_column_default_padding.bottom.size,
+            )))
+            .with(Modify::new(Columns::last()).with(Padding::new(
+                last_column_default_padding.left.size,
+                ZERO_PADDING,
+                last_column_default_padding.top.size,
+                last_column_default_padding.bottom.size,
+            )));
+        table
+    }
+
+    fn truncate_table_column(
+        mut table: Table,
+        column_position: usize,
+        remaining_terminal_width: usize,
+        suffix_additional_info: &str,
+    ) -> Table {
+        table.with(
+            Modify::new(Columns::single(column_position))
+                .with(Width::truncate(remaining_terminal_width).suffix(suffix_additional_info)),
+        );
+        table
+    }
+
+    fn wrap_table_column(
+        mut table: Table,
+        column_position: usize,
+        remaining_terminal_width: usize,
```

Comment:
Same here, shall I introduce a new type to prevent accidentially swapped arguments ?

Reply:
Done